pkg/buildpack/buildpack.go

```diff
@@ -29,6 +32,7 @@ type BuildModule interface {
 	// timestamp and root UID/GID).
 	Open() (io.ReadCloser, error)
 	Descriptor() Descriptor
+	ContainsFlattenedModules() bool
```

Comment:
This method is added to optimize when a BuildModule is going to be write into a tar, we need to know if there are more than one buildpack inside of it. Initially I was opening the tar and reading its content